wsi/drm/drmdisplay.cpp

```diff
@@ -248,8 +248,8 @@ bool DrmDisplay::GetDisplayConfigs(uint32_t *num_configs, uint32_t *configs) {
   size_t modes_size = modes_.size();
   SPIN_UNLOCK(display_lock_);
 
-  if (modes_size == 0) {
-    return PhysicalDisplay::GetDisplayConfigs(num_configs, configs);
+  if (modes_size == 0 || !IsConnected()) {
+    return false;
```

Comment:
This will break Android where it expects minimum values. We need to look this in more detail.